qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -37,26 +37,7 @@ You can download an example sample information file and prep information file fr
-| Value                                | Description                                                                                                                                      |
-+======================================+==================================================================================================================================================+
-| ``not applicable``                   | metadata information is inappropriate to report, can indicate that the standard itself fails to model or represent the information appropriately |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``not available: not collected``     | metadata information of an expected format was not given because it has not been collected                                                       |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``not available: to be reported``    | metadata information of an expected format was not given, a value will be given at the later stage                                               |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``not available: restricted access`` | metadata information exists but can not be released openly                                                                                       |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``obscured``                         | metadata information has been provided deliberately with low precision                                                                           |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``temporarily obscured``             | metadata information has been provided deliberately with low precision, a higher precision value will be provided at a later stage               |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-
-The last two, which consider level of precision, will be used, for example, for geo-references (latitude, longitude and country/sea).
+For the latest definitions visit the `EBI/ENA Missing value reporting <http://www.ebi.ac.uk/ena/about/missing-values-reporting>`__.
```

Comment:
I would say just a comma-separated list of the exact way to write the
terms, the table+description is already taken care of by EBI's
documentation, so I would say it's probably redundant to duplicate that
here.

On (Jun-30-16|12:18), Antonio Gonzalez wrote:
> > -+======================================+==================================================================================================================================================+
> > -| ``not applicable``                   | metadata information is inappropriate to report, can indicate that the standard itself fails to model or represent the information appropriately |
> > -+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
> > -| ``not available: not collected``     | metadata information of an expected format was not given because it has not been collected                                                       |
> > -+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
> > -| ``not available: to be reported``    | metadata information of an expected format was not given, a value will be given at the later stage                                               |
> > -+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
> > -| ``not available: restricted access`` | metadata information exists but can not be released openly                                                                                       |
> > -+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
> > -| ``obscured``                         | metadata information has been provided deliberately with low precision                                                                           |
> > -+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
> > -| ``temporarily obscured``             | metadata information has been provided deliberately with low precision, a higher precision value will be provided at a later stage               |
> > -+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
> > -
> > -The last two, which consider level of precision, will be used, for example, for geo-references (latitude, longitude and country/sea).
> > +For the latest definitions visit the `EBI/ENA Missing value reporting <http://www.ebi.ac.uk/ena/about/missing-values-reporting>`__.
> 
> As a table or just a list? 
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1874/files/7925b18d5f26b6eab295983a5e8a14eb72c4eb37#r69193637